modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -8,17 +8,17 @@
 
 	&,
 	& a {
-		color: var( --n-tabs-title-color-active );
+		color: var(--n-tabs-title-color-active);
```

Comment:
I believe we did that already?

<img width="1251" alt="Screenshot 2023-08-18 at 16 39 27" src="https://github.com/elementor/elementor/assets/94352322/653176eb-e394-4842-989c-99b6175e9c75">